README.md

```diff
@@ -46,111 +47,115 @@ See [action.yml](action.yml) for all options.
 
-This behavior is the recommended workflow for a GitHub repository that contains Protobuf files.
-However, the action can also be configured in a number of ways to match your preferred workflow.
+### Skipping steps
 
-#### Skipping steps
-
-For specific project requirements you may want to skip certain checks in CI.
-By default this action uses labels on pull requests to enable skipping of lint, formatting or breaking change detection checks.
-To use this behaviour you'll need to add the following labels to your repository:
+The default configuration makes it possible to skip lint, formatting, or breaking change checks on a PR
+by adding a label with a (case-insensitive) special name to that PR.
 
+- `buf skip breaking`: skips breaking change detection.
 - `buf skip lint`: skips lint.
 - `buf skip format`: skips format. 
-- `buf skip breaking`: skips breaking change detection.
 
-When a label is applied to a PR the action will re-run skipping the check specified by the label.
-All label checks are case-insensitive.
-Ensure the workflow file includes the types `labeled` and `unlabeled` events to automatically re-run on label changes.
+Ensure the workflow file includes the `pull_request` event types `labeled` and `unlabeled` so checks re-run on label changes.
+
+<!-- show how to disable the ability to allow these checks to be skipped? --> 
 
 ### Versioning
 
 To ensure the version of `buf` is consistent across workflows it's recommended to always use an explicit version.
 
 ```yaml
-- uses: bufbuild/buf-action@v0.1.3
+- uses: bufbuild/buf-action@v0.1
   with:
     version: 1.32.2
 ```
 
 If no version is specified in the workflow config, the action will resolve the version in order of precendence:
 - A version specified in the environment variable `${BUF_VERSION}`.
-- The version of `buf` that is already installed on the runner (if it exists)
+- The version of `buf` that is already installed on the runner (if it exists).
 - The latest version of the `buf` binary from the official releases on GitHub.
 
 ### Authentication
 
-Syncing the repository with the Buf Schema Registry (BSR) provides a seamless experience from your GitHub sources to your consumers of your schemas.
-Authenticating with the BSR is required for both push and label archive steps.
+[Publishing schemas](https://buf.build/docs/bsr/module/publish) to the Buf Schema Registry (BSR) provides a seamless
+way for consumers of your APIs to generate code.
+Authenticating with the BSR is required for both the push and archive label steps.
 
 To authenticate with the BSR, set the inputs `username` and `token`.
-The `username` and `token` values can be stored as secrets in the repository settings.
+The `username` and `token` values should be stored as secrets in the repository settings.
 The `token` value can be [generated from the Buf Schema Registry UI](https://buf.build/docs/bsr/authentication#create-an-api-token).
 
+> [!IMPORTANT]  
+> Never hardcode these values in the workflow file.
```

Comment:
Could link to the GitHub secrets? https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions

Reply:
done in dd0894631e4334185e0f9a376798c7df6a812319